src/window.rs

```diff
@@ -426,6 +517,35 @@ impl Window {
             WindowEvent::KeyboardInput { event, .. } => {
                 let event = keyboard_event_from_winit(event, self.modifiers_state.get());
                 log::trace!("Verso is handling {:?}", event);
+
+                /* Window operation keyboard shortcut */
```

Comment:
This should abstract to a helper method. I imagine we will have to deal with more shortcut in the future.

Reply:
I'll extract it as a function now and see if you have any ideas to make it more flexible.